packages/web/src/common/store/queue/sagas.ts

```diff
@@ -347,9 +333,11 @@ export function* watchNext() {
     }
 
     const id = (yield* select(getQueueTrackId)) as ID
+    const playerBehavior = (yield* select(getPlayerBehavior) || undefined) as
```

Comment:
why is the `|| undefined` necessary?

Reply:
it can be null otherwise and the fn we're passing to doesn't like that. We could do it below instead, but 🤷

Reply:
would prefer to do `?? undefined` but nitty. both are kinda ick.

Reply:
but at least then you'd avoid the explicit typing